clients/drcachesim/common/trace_entry.h

```diff
@@ -431,6 +431,12 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_RSEQ_ENTRY,
 
```

Comment:
Update docx of TRACE_MARKER_TYPE_RSEQ_ABORT to specify what the address in the marker represents.

Reply:
Done.